bh-sd-jwt/CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ Versioning](https://doc.rust-lang.org/cargo/reference/semver.html).
 
 ## [Unreleased]
 
+## [0.2.0] - 2025-06-17
```

Comment:
You also need to update the link to `[0.2.0]` at the bottom of CHANGELOG.md, currently it is missing.